zerver/migrations/0231_userprofile_avatar_hash.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-06-27 21:10
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0230_rename_to_enable_stream_audible_notifications'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='userprofile',
+            name='avatar_hash',
+            field=models.CharField(max_length=120, null=True),
```

Comment:
How did you pick 120 as the length?

Reply:
I actually chose a random/safe number for this. I wanted to confirm what to choose for the final version.

Reply:
We should use the length of the strings generated by the hash function.

Reply:
Got it. I think it was ~110, I'll check again and make the change.